src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -239,13 +239,8 @@ def launch_fluent(
         default is ``True``. You can stop and start the streaming of the
         Fluent transcript subsequently via the method calls, ``transcript.start()``
         and ``transcript.stop()`` on the session object.
-    ui_mode : UIMode or str, optional
-        Fluent user interface mode. Options are either the values of the ``UIMode``
-        enum or any of ``"no_gui_or_graphics"``, ``"no_gui"``, ``"hidden_gui"``,
-        ``"no_graphics"`` or ``"gui"``. The default is ``UIMode.HIDDEN_GUI`` in
-        Windows and ``UIMode.NO_GUI`` in Linux. ``"no_gui_or_graphics"`` and
-        ``"no_gui"`` user interface modes are supported in Windows starting from Fluent
-        version 2024 R1.
+    ui_mode : UIMode
+        Defines the user interface mode for Fluent. Options correspond to values in the ``UIMode`` enum.
```

Comment:
@hpohekar

Should we still mention that `ui_mode` also accepts `str` values? The type hint allows it, but the new docstring implies only `UIMode` is valid. If deprecation of string values isn’t intended, maybe we could clarify with something like:

```python
    ui_mode : UIMode or str, optional  
        Defines the user interface mode for Fluent. Accepts either a ``UIMode`` value  
        or a corresponding string such as ``"no_gui"``, ``"hidden_gui"``, or ``"gui"``.
```